fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -1683,6 +1687,11 @@ private PlanNode createScanNode(Analyzer analyzer, TableRef tblRef, SelectStmt s
 
         switch (tblRef.getTable().getType()) {
             case OLAP:
+                if (tblRef.isMetaScan()) {
+                    MetaScanNode metaScanNode = new MetaScanNode(ctx_.getNextNodeId(), tblRef.getDesc());
```

Comment:
if we only could do meta scan on olap table, should we named this node to `OlapMeataScanNode` ?

Reply:
done